yoti_python_sdk/protobuf/protobuf.py

```diff
@@ -40,13 +43,20 @@ def anchor(data):
         return anchor
 
     def value_based_on_content_type(self, value, content_type=None):
-        if content_type == self.CT_UNDEFINED:
-            raise TypeError('Wrong content type')
-        elif content_type == self.CT_STRING:
+        if content_type == self.CT_STRING:
             return value.decode('utf-8')
         elif content_type == self.CT_DATE:
             return value.decode('utf-8')
-        return value
+        elif content_type == self.CT_JPEG \
+                or content_type == self.CT_PNG:
+            return value
+        elif content_type == self.CT_INT:
+            string_value = value.decode('utf-8')
+            int_value = int(string_value)
+            return int_value
```

Comment:
Could this be `return int(value.decode('utf-8'))`? _(probably clearer as you have it)_

Reply:
I do think it gives more clarity if we separate it